docs/_interface/sync-modes.md

```diff
@@ -0,0 +1,118 @@
+There are some hardware factors that determine the speed of the state healing (speed of disk read/write and internet 
+connection) and also the total gas used in each block (more gas means more changes to the state that have to be 
+handled).
+
+**Note** Snap sync is the default behaviour, so if the `--syncmode` value is not passed to Geth at startup, 
+Geth will use snap sync. A node that is started using `snap` will switch to block-by-block sync once it has 
+caught up to the head of the chain.
+
+### Full
+
+A full sync generates the current state by executing every block starting from the genesis block. A full sync 
+indendently verifies proof-of-work and block provenance as well as all state transitions by re-executing the 
+transactions in the entire historical sequence of blocks. Only the most recent 128 blocks are stored in a full 
+node - older blocks are pruned periodically and represented as a series of checkpoints from which any previous 
+state can be regenerated on request. 128 blocks is about 25.6 minutes of history with a block time of 12 seconds. 
+To create a full node pass `--syncmode full` at startup.
+
+## Archive nodes
+
+An archive node is a node that retains all historical data right back to genesis. There is no need to regenerate 
+any data from checkpoints because all data is directly available in the node's own storage. Archive nodes are 
+therefore ideal for making fast queries about historical states. At the time of writing (September 2022) a full 
+archive node that stores all data since genesis occupies nearly 12 TB of disk space (keep up with the current 
+size on [Etherscan](https://etherscan.io/chartsync/chainarchive)). Archive nodes are created by configuring Geth's 
+garbage collection so that old data is never deleted: `geth --syncmode full --gcmode archive`. 
+
+It is also possible to create a partial/recent archive node where the node was synced using `snap` but the state 
+is never pruned. This creates an archive node that saves all state data from the point that the node first syncs. 
+This is configured by starting Geth with `--syncmode snap gcmode archive`.
+
+## Light nodes
+
+A light node syncs very quickly and stores the bare minimum of blockchain data. Light nodes only process block
+headers, not entire blocks. This greatly reduces the computation time, storage and bandwidth required relative to a 
+full node. This means light nodes are suitable for resource-constrained devices and can catch up to the head of the
+chain much faster when they are new or have been offline for a while. The trade-off is that light nodes rely heavily 
+on data served by altruistic full nodes. A light client can be used to query data from Ethereum and submit transactions, 
+acting as a locally-hosted Ethereum wallet. However, because they don't keep local copies of the Ethereum state, light 
+nodes can't validate blocks in the same way as full nodes - they have to trust that the data they receive is honest. 
```

Comment:
Light client does sacrifice some security guarantee, but it's not totally trust the received data. Instead, full node needs to attach a proof in the responded packet and light client can verify it against the local header chain.